db/migrate/20151106172531_remove_uniqueness_display_name_index_users.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveUniquenessDisplayNameIndexUsers < ActiveRecord::Migration
+  def change
+    remove_index :users, column: :display_name, unique: true
```

Comment:
Dont we still want display names to be unique? What happens if I start impersonating moderators by setting my display name the same?